extensions/open-in-shopify-admin/src/open-in-shopify-admin.ts

```diff
@@ -0,0 +1,33 @@
+import { closeMainWindow, getFrontmostApplication, showToast, Toast } from "@raycast/api";
+import { getBrowserURL, openURL } from "./utils";
+
+const command = async () => {
+  await closeMainWindow();
+  await showToast({ title: "Opening in Shopify admin...", style: Toast.Style.Animated });
+
+  const frontmostApp = await getFrontmostApplication();
+
+  if (frontmostApp.bundleId !== "com.apple.finder") {
```

Comment:
Is this the best implementation? Looks a bit funny

Reply:
Actually this check was redundant and not needed since we do check if browser URL is there and show a error message if the user tries to run the extension on a normal window

Reply:
https://github.com/user-attachments/assets/2b2acdb5-a6e0-4516-a549-33a1fad239a0